app/controllers/admin/users_controller.rb

```diff
@@ -50,8 +66,11 @@ def edit; end
     private
 
     def user_params
-      params.require(:user).permit(:email, :name, :email_public, :biography, :nickname, :affiliation, :is_admin, :username, :login, :is_disabled,
-                                   :tshirt, :mobile, :volunteer_experience, :languages, :to_confirm, role_ids: [])
+      permit_base = [:email, :name, :email_public, :biography, :nickname, :affiliation, :is_admin, :username, :login, :is_disabled,
+                     :tshirt, :mobile, :volunteer_experience, :languages, :to_confirm, role_ids: []]
+      # no password for ICHAIN-enabled mode
+      permit_base << :password unless ENV['OSEM_ICHAIN_ENABLED'] == 'true'
```

Comment:
As this is the admin controller I would just add the ``:password`` to the permit without the conditional.